demos/benchmark/embeddings/benchmark_embeddings.py

```diff
@@ -165,7 +260,23 @@ async def limited_request_func(request_func_input, pbar):
     }
     return result
 
-benchmark_results = asyncio.run(benchmark(docs=docs, model=args["model"], api_url=args["api_url"],request_rate=float(args["request_rate"])))
+if args["backend"] == "ovms-embeddings":
+    backend_function = async_request_embeddings
+    default_api_url = "http://localhost:8000/v3/embeddings"
+elif args["backend"] == "ovms_rerank":
+    backend_function = async_request_rerank
+    default_api_url = "http://localhost:8000/v3/rerank"
+elif args["backend"] == "tei-embed":
+    backend_function = async_request_embeddings_tei
+    default_api_url = "http://localhost:8080/embed"
+elif args["backend"] == "infinity-embeddings":
+    backend_function = async_request_embeddings
+    default_api_url = "http://localhost:7997/embeddings"
```

Comment:
"else" can be added with exception raised, when backed is different than expected